docs/source/user_guide/basic.rst

```diff
@@ -180,34 +181,39 @@ We can also use :meth:`cupy.ndarray.get()`:
 Memory management
 -----------------
 
-Check :doc:`./memory` for a detailed description of how is memory managed in CuPy
+Check :doc:`./memory` for a detailed description of how memory is managed in CuPy
 using memory pools.
 
 
-How to write CPU/GPU agnostic code
+How to write device agnostic code
```

Comment:
I'd prefer the "CPU/GPU agnostic" terminology in this section to remain unchanged. "Device agnostic" has a different meaning, usually associated with code portability (ex; vendor-neutral support such as the support of NVIDIA/AMD GPUs). We should add a section on this point in a separate PR.

Reply:
Thanks for the feedback/comments, @leofang. I agree with all of them except this one. I understand that saying "device agnostic" is not exactly right and that the word "device" is primarily used to mean GPUs, but I feel that saying "CPU/GPU agnostic code" is a bit misleading. The slash usually means "or", so if we consider only one part (say, CPU) -- it would become "CPU agnostic", which implies that code is agnostic of CPU it runs on (which is, probably, true, but not what this section is about). Should it be "CPU-GPU agnostic", perhaps? I'm OK leaving this "as-was" but it actually confused me when I was reading it, hence the PR.

Reply:
I reverted this change (and changed the next paragraph accordingly) but I suggest to take a closer look at this at some point in the future.

Reply:
> The slash usually means "or", so if we consider only one part (say, CPU) -- it would become "CPU agnostic", which implies that code is agnostic of CPU it runs on (which is, probably, true, but not what this section is about). Should it be "CPU-GPU agnostic", perhaps? I'm OK leaving this "as-was" but it actually confused me when I was reading it, hence the PR.

Oh, OK, so this was what confused you. I'd say it's kinda a common term used in the field --- If you google it, you'd find way more entries using "CPU/GPU agnostic" than using "CPU-GPU agnostic", although a portion of those are focusing on CuPy/Chainer 😄 

Not that a forward slash has overloaded meanings in English --- in addition to "or", it can also be used for punctuation, indicating connection/conflict, or **"and"**. (See, I just said "CuPy/Chainer".)